content/en/data_jobs/databricks.md

```diff
@@ -19,23 +19,21 @@ Follow these steps to enable Data Jobs Monitoring for Databricks.
 
 1. [Configure the Datadog-Databricks integration](#configure-the-datadog-databricks-integration) with your Databricks API token.
 1. [Install the Datadog Agent](#install-the-datadog-agent-on-your-databricks-clusters) on your Databricks cluster(s).
-1. [Add your Datadog API key](#add-your-datadog-api-key-in-databricks) in Databricks.
 
-### Configure the Datadog-Databricks integration 
+### Configure the Datadog-Databricks integration
 
 1. In your Databricks workspace, go to **Settings** > **Developer**. Next to **Access tokens**, click **Manage**.
 1. Click **Generate new token**, enter a comment, and click **Generate**. Take note of your token.
 1. In Datadog, open the Databricks integration tile.
-1. On the **Configure** tab, click **Add New**. 
+1. On the **Configure** tab, click **Add New**.
 1. Enter a workspace name, your Databricks workspace URL, and the Databricks token you generated.
    {{< img src="data_jobs/databricks/configure-token.png" alt="In the Datadog-Databricks integration tile, a Databricks workspace is displayed. This workspace has a name, URL, and API token." style="width:100%;" >}}
 
 
 ### Install the Datadog Agent on your Databricks cluster(s)
 
-Use one of the following init scripts:
+Use the following init script:
 - [Install the Datadog Agent][8]
-- [Install the Datadog Agent without logs][2]
 
 You can choose to install the Agent globally, or on a specific Databricks cluster.
 
```

Comment:
I found this part a bit confusing – I first went to the Compute section on the Databricks side nav not the Admin Settings. Maybe the steps can be stated more explicitly?

1. In Databricks, click your **display name** (email address) in the upper right corner of the page.
1. Select **Admin Settings**.
1. In the Settings list of sections, select **Workspace Admin** > **Compute**.
1. In the **All purpose clusters** section, next to **Global init scripts**, click **Manage**.
1. Click **Add**. Name your script. Then, in the **Script** field, copy and paste the init script [`datadog_databricks_job_monitoring_init.sh`][8]. 
...

Reply:
I agree those might not be very straightforward and Databricks is not very verbose about it neither https://docs.databricks.com/en/init-scripts/global.html#add-a-global-init-script-using-the-ui

Added a slight change from `Settings` -> `Admin Settings` but do you think we should more descriptive than databricks ?

Reply:
I like Joe's suggestion. Let's be more explicit to help users who might not be as familiar with the Databricks console.